.travis.yml

```diff
@@ -12,6 +12,7 @@ php:
 before_script:
   - if [ "`phpenv version-name`" != "hhvm" ]; then echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi
   - if [ "`phpenv version-name`" != "hhvm" ]; then echo "extension = amqp.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi
+  - composer self-update
```

Comment:
sometimes 30-60 days is not enough :) today test was failing for this PR because of old composer version